src/components/AppProvider/utilities/withSticky/withSticky.tsx

```diff
@@ -1,55 +0,0 @@
-import * as React from 'react';
```

Comment:
A smaller more compact version was included in `withAppProvider` so we can cut down on individual `HoC`